src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/GetRecallComponentTest.kt

```diff
@@ -208,6 +208,7 @@ class GetRecallComponentTest : ComponentTestBase() {
       .jsonPath("$.seniorProbationOfficerInfo.fullName").isEqualTo(fullyPopulatedRecall.seniorProbationOfficerInfo!!.seniorProbationOfficerName.value)
       .jsonPath("$.seniorProbationOfficerInfo.email").isEqualTo(fullyPopulatedRecall.seniorProbationOfficerInfo!!.seniorProbationOfficerEmail.value)
       .jsonPath("$.seniorProbationOfficerInfo.phoneNumber").isEqualTo(fullyPopulatedRecall.seniorProbationOfficerInfo!!.seniorProbationOfficerPhoneNumber.value)
+      .jsonPath("$.seniorProbationOfficerInfo.probationEmail").isEqualTo(fullyPopulatedRecall.seniorProbationOfficerInfo!!.seniorProbationFunctionalEmail.value)
```

Comment:
I don't get the name; IMO there's more information in `seniorProbationOfficerInfo.functionalEmail` versus repeating `probation` in `seniorProbationOfficerInfo.probationEmail`?